package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "ember-cli-babel": "^7.1.2",
-    "ember-concurrency": "^0.8.27",
+    "ember-concurrency": "^0.8.27 || ^0.9.0 || ^0.10.0",
```

Comment:
Would `~0.8.27` be a simpler and longer solution to this?

Reply:
I think that will restrict it further to only versions within 0.8.x instead of allowing it to be a higher version

Reply:
0.x versions are kinda weird in that using ^ will only bump the third number since below version 0 considers the second number to be semver major

Reply:
Yeah, you're totally right. I 100% spaced on this. Maybe something like this `>=0.8.0 < 1.0.0` ?

Reply:
i think that could work the only downside might be if theres a version 0.11.0 and it has a breaking change thats incompatible with this library

Reply:
Alright, I'll just merge this and patch release it. I can always come back to it 😄